src/cmd/flux-job.c

```diff
@@ -391,6 +406,91 @@ int cmd_raise (optparse_t *p, int argc, char **argv)
     return 0;
 }
 
+/*
+ * List generated by:
+ *
+ * $ kill -l | sed 's/[0-9]*)//g' | xargs -n1 printf '    "%s",\n'
+ *
+ * (ignoring SIGRT*)
+ */
+static const char *sigmap[] = {
```

Comment:
Duplicate of `sys_siglist[]`?  (see psignal(3))

Reply:
oh duh, not!  (forehead slap)

Reply:
I did feel this was a bit hacky, and should at least wrap each definition in `#ifdef SIGXXX` like is done in coreutils `kill.c` -- however, I felt maybe this was ok for now and could be "fixed" later?

Reply:
Agreed.